atomisticparsers/gromacs/parser.py

```diff
@@ -397,6 +396,7 @@ def get_force_field_parameters(self, gromacs_version: str = None) -> List[Dict]:
             )
             return []
         gromacs_version = gromacs_version.split('.')[0] if gromacs_version else None
+        # ! This warning does not get triggered, why?
```

Comment:
Can we try to address this comment before merging? Is the version actually "2024" or it's something like "2024.1".

If it's necessary to leave the comment then at least add some info so it is clear what exactly what context the comment is referrring

Reply:
No, this is an old comment. An earlier error prevented the code from reaching this check at some point. I removed the comment now.